### PR TITLE
doc: Bip70 removal follow-up

### PR DESCRIPTION
**Pull request by fjahr**
A small follow-up to #17165 which removed BIP70 support.

1. Removes one leftover mention of BIP70 in a comment.
2. Removes BIP70 reference in comments on network/chain name strings. These can be removed as they are not really helpful and also incorrect: BIP70 only defines "main" and "test" but not "regtest". If/When signet gets merged we will add another name to the list that is not defined in BIP70. Mostly there is also an exhaustive list of the options included in the comment anyway.

If we would like to keep an identifier for this naming scheme, I would suggest switching to something more generic, like 'short chain name'. Happy to implement that if that is preferred. Alternatively, we could add a reference to `CBaseChainParams`. That would also mean we don't have to change these lines again for signet.

### CODE REVIEW

**Comment by laanwj** [comment]
> Removes BIP70 reference in comments on network/chain name strings

I was hesitant about this at first, but tend to agree nothing is lost by simply calling it "chain name". They are part of the RPC API so this doesn't mean it can be changed willy-nilly from now on, though.

**Comment by MarcoFalke** [comment]
Concept ACK

File: src/qt/walletmodel.h [hunk 1]
```diff
@@ -64,7 +64,7 @@ class SendCoinsRecipient
     CAmount amount;
     // If from a payment request, this is used for storing the memo
     QString message;
-    // If building with BIP70 is disabled, keep the payment request around as
+    // The payment request needs to be kept around as
```
Comment (jonatack) [inline src/qt/walletmodel.h]: Suggestion:
```cpp
    // Keep the payment request around as a serialized string to ensure
    // load/store is lossless.
```

File: src/qt/walletmodel.h [hunk 2]
```diff
@@ -64,7 +64,7 @@ class SendCoinsRecipient
...
     // serialized string to ensure load/store is lossless
```
Comment (jonatack) [inline src/qt/walletmodel.h]: Suggestion:
```cpp
    // Keep the payment request around as a serialized string to ensure
    // load/store is lossless.
```

**Review by jonatack — COMMENTED** [review]
ACK 3f5be66b6c4466a72c2409fe53d2a2da01c87702

Feel free to ignore the suggestion below. Note that since all changes are documentation ones, the commit prefixed with "wallet" could be amended to "doc".

File: src/qt/walletmodel.h (hunk 2 above)
Comment (fjahr) [reply src/qt/walletmodel.h]: Took your suggestion. Thanks!

**Comment by jachiang** [comment]
ACK b413bdf6b

File: src/util/system.h [hunk 3]
```diff
@@ -265,7 +265,7 @@ class ArgsManager
     void ForceSetArg(const std::string& strArg, const std::string& strValue);
 
     /**
-     * Looks for -regtest, -testnet and returns the appropriate BIP70 chain name.
+     * Looks for -regtest, -testnet and returns the appropriate chain name.
```
[the remark this answers is not in the document]
Comment (fjahr) [reply src/util/system.h]: done

File: src/chainparamsbase.h [hunk 4]
```diff
@@ -15,7 +15,7 @@
 class CBaseChainParams
 {
 public:
-    /** BIP70 chain name strings (main, test or regtest) */
+    /** Chain name strings (main, test or regtest) */
     static const std::string MAIN;
```
[the remark this answers is not in the document]
Comment (fjahr) [reply src/chainparamsbase.h]: done, with grouping

File: src/chainparams.h [hunk 5]
```diff
@@ -75,7 +75,7 @@ class CChainParams
     uint64_t AssumedChainStateSize() const { return m_assumed_chain_state_size; }
     /** Whether it is possible to mine blocks on demand (no retargeting) */
     bool MineBlocksOnDemand() const { return consensus.fPowNoRetargeting; }
-    /** Return the BIP70 network string (main, test or regtest) */
+    /** Return the network string (main, test or regtest) */
```
[the remark this answers is not in the document]
Comment (fjahr) [reply src/chainparams.h]: done

**Comment by MarcoFalke** [comment]
ACK 3ed8e3d079a3860dcdf944f7c1aa37765a53da32

**Pull request merged**